### PR TITLE
update: src/content/Docs/providers/akash-provider-feature-discovery-up 

### PR DESCRIPTION
**Pull request by HoomanDgtl**
#139

### CODE REVIEW

File: src/content/Docs/providers/build-a-cloud-provider/gpu-resource-enablement/index.md [hunk 1]
```diff
@@ -170,6 +171,30 @@ source venv/bin/activate
...
 
+## Enabling GPU Resources on Akash
+
+To set up GPU resources on Akash, providers need to ensure their configuration allows for proper GPU detection. This guide will walk you through essential resources, troubleshooting steps, and configuration tips to get your GPUs recognized and operational.
+
+### 1. Important Guides for GPU Configuration
+
+For a comprehensive setup, new providers should start with these foundational guides:
+
+- [Provider Feature Discovery GPU Configuration Integration Guide](/docs/providers/provider-feature-discovery-gpu-configuration-integration-guide/): Step-by-step instructions for configuring GPUs to work seamlessly with Akash.
```
Comment (vigneshv-ocl) [inline src/content/Docs/providers/build-a-cloud-provider/gpu-resource-enablement/index.md:182]: the URL leads no where. i get a `404 - page not found`

File: src/content/Docs/providers/build-a-cloud-provider/gpu-resource-enablement/index.md [hunk 2]
```diff
@@ -170,6 +171,30 @@ source venv/bin/activate
...
+- [Akash Provider Feature Discovery Upgrade Enablement](/docs/providers/akash-provider-feature-discovery-upgrade-enablement/): Detailed guidance on necessary upgrades to fully support GPU functionality.
```
Comment (vigneshv-ocl) [inline src/content/Docs/providers/build-a-cloud-provider/gpu-resource-enablement/index.md:183]: the URL leads no where. i get a 404 - page not found

Comment (vigneshv-ocl) [reply src/content/Docs/providers/build-a-cloud-provider/gpu-resource-enablement/index.md:183]: can you also fix the link in L502

**Comment by HoomanDgtl** [comment]
Hii @vigneshv-ocl , it's working: maybe refer to this live pr here: https://akashnetworkwebsite.vercel.app/docs/providers/build-a-cloud-provider/gpu-resource-enablement/#enabling-gpu-resources-on-akash

**Review by vigneshv-ocl — APPROVED** [review]

**Comment by vigneshv-ocl** [comment]
> Hii @vigneshv-ocl , it's working: maybe refer to this live pr here: https://akashnetworkwebsite.vercel.app/docs/providers/build-a-cloud-provider/gpu-resource-enablement/#enabling-gpu-resources-on-akash

thanks , approved

**Review by troian — APPROVED** [review]

**Pull request merged**